macosx/InfoActivityViewController.mm

```diff
@@ -180,50 +179,68 @@ - (CGFloat)vertLayoutHeight
     return NSHeight(self.fTransferView.frame) + NSHeight(self.fDatesView.frame) + (2 * kStackViewInset) + kStackViewVerticalSpacing;
 }
 
-- (CGFloat)changeInWindowHeight
+- (void)updateOldHeight:(CGFloat)height
+{
+    self.oldHeight = height;
+}
+
+- (CGFloat)heightChange
 {
-    CGFloat difference = 0;
+    return self.oldHeight - self.currentHeight;
+}
 
+- (NSRect)viewRect
+{
+    CGFloat difference = self.heightChange;
+
+    NSRect windowRect = self.view.window.frame, viewRect = self.view.frame;
```

Comment:
We generally prefer one assignment per line, please.

Reply:
sure thing, I cleaned up some other declarations also